_opensearch/remote.md

```diff
@@ -11,6 +11,51 @@ Remote-backed storage is an experimental feature. Therefore, we do not recommend
 
 Remote-backed storage offers OpenSearch users a new way to protect against data loss by automatically creating backups of all index transactions and sending them to remote storage. In order to expose this feature, segment replication must also be enabled. See [Segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/) for additional information.
 
+## Translog
+
+When documents are indexed, updated, or deleted, the data needs to eventually be persisted to disk so it can be recovered in case of failure. However, Lucene commits are expensive operations, and as such cannot be performed after every change to the index. Instead, each shard records every indexing operation in a transaction log called *translog*. When a document is indexed, it is added to the memory buffer and also recorded in the translog. Frequent refresh operations write the documents in the memory buffer to a segment, and clear the memory buffer. Periodically, a flush performs a Lucene commit, which includes writing the segments to disk using `fsync`, purging the old translog, and starting a new translog. 
+
+Thus, a translog contains all operations that have not yet been written to disk. 
+
+## Segment replication and remote-backed storage
+
+When neither segment replication nor remote-backed storage is enabled, OpenSearch uses document replication. In document replication, when a write request lands on the primary shard, the request is indexed to Lucene and stored in the translog as a single unit of transaction for durability. After this, the request is sent to the replicas, where, in turn, it is indexed to Lucene and stored in the translog. 
```

Comment:
we can get rid of `as a single unit of transaction for durability` for the 2nd sentence. For the 3rd sentence, at the end we can add `for durability`.